apps/condo/domains/organization/schema/AcceptOrRejectOrganizationInviteService.test.js

```diff
@@ -12,7 +12,11 @@ describe('AcceptOrRejectOrganizationInviteService', () => {
         const client1 = await makeClientWithRegisteredOrganization()
         const client2 = await makeClientWithNewRegisteredAndLoggedInUser()
 
-        const [invite] = await inviteNewOrganizationEmployee(client1, client1.organization, client2.userAttrs)
+        const [invite] = await inviteNewOrganizationEmployee(client1, client1.organization, client2.userAttrs, {
```

Comment:
it the role is require it's common to add it as new required argument to inviteNewOrganizationEmployee functioin

Reply:
Yes, I wanted to do it in another PR

Reply:
Oops, another commits has slipped in my PR.

Reply:
Rebased

Reply:
`role` mutation variable was made required in another PR #572